test/util/compareHtml.js

```diff
@@ -113,6 +113,8 @@ const BLOCK_NODES = [
     'figure',
     'list',
     'media',
+    'media_alt',
+    'media_text',
```

Comment:
`media_alt` and `media_text` are Slate nodes and not CNXML elements, and as such they will never occur in CNXML. `div` on the other hand will and should be included in `BLOCK_NODES`.